docs/overview/diagrams.md

```diff
@@ -121,6 +121,45 @@ classDiagram
 
+# AssociativeEither
+
+[![AssociativeCompose hierarchy][AssociativeEither-image]][AssociativeEither-link]
+
+```mermaid
+classDiagram
+  AssociativeEither~F<_>~ <|-- CommutativeEither~F<_>~
+  AssociativeEither~F<_>~ <|-- IdentityEither~F<_>~
+  class AssociativeEither~F<_>~{
```

Comment:
There are a couple of instances which are already `CommutativeEither`, like Future or ZIO, so we don't need to list them here in `AssociativeEither`.
It's already implied that they are `AssociativeEither` just by being `CommutativeEither`.

Reply:
To expand on my point above: The fact that there are instances (like those for the like of ZIO or Future) that are implemented both in `AssociativeEither`'s companion object and `CommutativeEither`'s companion object is essentially a bug in ZIO Prelude's design.
The instances should be only in the `AssociativeEither`'s companion object. But we'll have to deal with that in another PR.

Reply:
@sideeffffect 
I would like to address the bug that you mentioned before, could you please create the issue with his respective description and information in order to understand and solve this issue.

Reply:
That's the spirit! :muscle: 
Feel free to claim it https://github.com/zio/zio-prelude/issues/431